kingfisher_scrapy/base_spiders/big_file_spider.py

```diff
@@ -29,7 +30,16 @@ def start_requests(self):
 
     resize_package = True
 
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        spider = super(BigFileSpider, cls).from_crawler(crawler, *args, **kwargs)
+
+        if not spider.data_type or spider.data_type not in ('release_package', 'record_package'):
```

Comment:
This is simpler and has the same outcome.

```suggestion
        if spider.data_type not in ('release_package', 'record_package'):
```